src/StatusQ/Controls/StatusBaseInput.qml

```diff
@@ -0,0 +1,132 @@
+import QtQuick 2.14
+
+import StatusQ.Controls 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+
+Item {
+    id: statusBaseInput
```

Comment:
as per QML guidelines, it is recommended to name the root entity as "root".

Reply:
We've had this discussion before, in fact most of the components used `root` but eventually we decided to go for component names. Lemme see if I can find that discussion

Reply:
Can't find it anymore, it's somewhere in one of the PRs. Anyways, I think @borismelnik mentioned that not using `root` is preferable.

I personally don't have a strong opinion about that. Question is whether we gain anything from renaming all of them again (other than increasing chances to introduce bugs)

Reply:
i never see this recommendation - could you pls give me a link?

Reply:
i have this one https://doc.qt.io/qt-5/qml-codingconventions.html

Reply:
https://github.com/Furkanzmc/QML-Coding-Guide#cs-5-giving-components-ids

> Make sure that the top most component in the file always has root as its id. Qt will make unqualified name look up deprecated in QML 3, so it's better to start giving IDs to your components now and use qualified look up.